devito/finite_differences/coefficients.py

```diff
@@ -217,27 +227,41 @@ def generate_subs(i):
         return rules
 
 
+def extract_dim(dim):
```

Comment:
I think this should be moved into `symbolics/insepction.py` as a `retreive_dimension`

Reply:
Done